test/integration/select_test.lua

```diff
@@ -174,7 +178,7 @@ add('test_select_all', function(g)
     t.assert_equals(objects, customers)
 
     -- after obj 2
-    local after = customers[2]
+    local after = crud_utils.flatten(customers[2], g.space_format)
```

Comment:
`local after = result.rows[2]`

Reply:
I don't like it - in this case we depends on previous result.

Reply:
Well, ok...

But I see nothing bad